prac_05/emails.py

```diff
@@ -14,7 +14,9 @@ def main():
 
 
 def get_name_from_email(email):
-    name = email
+    prefix = email.split('@')[0]
+    parts = prefix.split('.')
```

Comment:
Can also make this a one liner if you wanted to:
```parts = email.split("@")[0].split(".")```

Reply:
Thanks. I agree with this.